### PR TITLE
Added install script and run script

### PR DESCRIPTION
**Pull request by wesleymonte**

### CODE REVIEW

File: install.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    url = "https://github.com/emanueljoivo/iwi/archive/{}.zip".format(WEB_UI_BRANCH)
+    download_repository(url)
+
+    iwi_dir_name = "iwi-{}".format(WEB_UI_BRANCH.replace("/", "-"))
+    os.chdir(iwi_dir_name)
+    os.system("npm install")
+    os.chdir("..")
+    return iwi_dir_name
+
+oauth_storage_service_token_url="oauth_storage_service_token_url"
```
Comment (emanueljoivo) [inline install.py]: The name of these properties on the .env file is a bit different. Please, verify in your local version the correct properties names for the Web UI.

Comment (emanueljoivo) [reply install.py]: For example, for the client identification, the property has the name`VUE_APP_OWNCLOUD_OAUTH_CLIENT_ID`.

**Comment by emanueljoivo** [comment]
You did a great job. Just an adjustment and I will accept your contribution.

File: install.py (hunk 1 above)
Comment (wesleymonte) [reply install.py]: Done.

File: install.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+job_submission_monitor_period="job_submission_monitor_period"
+
+VUE_APP_EXTERNAL_LINK_DOCS="VUE_APP_EXTERNAL_LINK_DOCS"
+VUE_APP_EXTERNAL_LINK_GITHUB_REPO="VUE_APP_EXTERNAL_LINK_GITHUB_REPO"
+VUE_APP_EXTERNAL_LINK_ABOUT="VUE_APP_EXTERNAL_LINK_ABOUT"
+
+VUE_APP_OWNCLOUD_SERVER_HOST="VUE_APP_OWNCLOUD_SERVER_HOST"
+VUE_APP_OWNCLOUD_OAUTH_CLIENT_ID="VUE_APP_OWNCLOUD_OAUTH_CLIENT_ID"
+VUE_APP_OWNCLOUD_OAUTH_SECRET="VUE_APP_OWNCLOUD_OAUTH_SECRET"
+VUE_APP_OWNCLOUD_OAUTH_REDIRECT_URI="VUE_APP_OWNCLOUD_OAUTH_REDIRECT_URI"
```
Comment (emanueljoivo) [inline install.py]: Note that this conf property is the same as set by OAuth2 to `redirect_uri`. In this case, replace the name of this property and group it together with the other OAuth2 properties.

File: install.py [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    return arrebol_dir_name
+
+def install_web_ui():
+    print("---> Installing Web UI")
+    url = "https://github.com/emanueljoivo/iwi/archive/{}.zip".format(WEB_UI_BRANCH)
+    download_repository(url)
+
+    iwi_dir_name = "iwi-{}".format(WEB_UI_BRANCH.replace("/", "-"))
+    os.chdir(iwi_dir_name)
+    os.system("npm install")
```
Comment (emanueljoivo) [inline install.py]: Replace by `yarn install`.

File: install.py [hunk 4]
```diff
@@ -0,0 +1,150 @@
+import os
+from zipfile import ZipFile
+
```
Comment (emanueljoivo) [inline install.py]: Add a print that lists all the dependencies required.
- OpenJDK8;
- Maven;
- Node < 11;
- Yarn

File: install.py [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+    monitors_periods = get_monitors_periods()
+    confs = {**ds_properties, **host_addresses, **monitors_periods}
+
+    iguassu_dir_name = install_iguassu()
+    arrebol_dir_name = install_arrebol()
+    web_ui_dir_name = install_web_ui()
+
+    write_web_ui_conf(confs, web_ui_dir_name)
+    write_iguassu_conf(confs, iguassu_dir_name)
+
```
Comment (emanueljoivo) [inline install.py]: It a good idea clear all the temporary both directories and zip files at the finish of the script or in any case of error, maybe is the case of use `try-finally` statements.

File: install.py (hunk 4 above)
Comment (wesleymonte) [reply install.py]: Done.

File: install.py (hunk 3 above)
Comment (wesleymonte) [reply install.py]: Done.

**Pull request merged**